### ISSUE TITLE
CPU usage graph goes off the top of the panel when 100% usage

### ISSUE THREAD

**Issue by mtrolley** [cjbassi/ytop#35]
ytop version 0.4.1
Linux ugc14vr4dh2 4.15.0-76-generic #86-Ubuntu SMP Fri Jan 17 17:24:28 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux
Gnome terminal
tmux 2.6

When CPU usage is 100% the graph goes outside the top of the CPU Usage panel so you don't see it. See attached screenshot.
![Screenshot from 2020-02-05 09-24-16](https://user-images.githubusercontent.com/1136971/73850783-5dd72b80-47fa-11ea-89c0-62aa64bec580.png)

**Comment by cjbassi** [comment]
Yeah, it seems like the whole graph is shifted upwards so we got to figure out what's causing that.

**Comment by ghost** [comment]
As a side qeustion can you disable the graphs ? Im going for a minimalistic look and i would prefer to not have them.

**Comment by cjbassi** [comment]
@linxed Yeah, I'd like to have some configurability around that. Could you open a separate issue for that though?
